src/native/libs/System.Globalization.Native/pal_locale.m

```diff
@@ -5,6 +5,9 @@
 #include "pal_locale_internal.h"
 
 #import <Foundation/Foundation.h>
+#import <Foundation/NSFormatter.h>
+
+#include "pal_localeStringData.h"
```

Comment:
Feels like this would be better to be right under `#include "pal_locale_internal.h"`. Is this needed if `pal_locale_internal.h` declares both `NativeGetLocaleName` and `NativeGetLocaleInfoString`? Or, what would the reason to not include `pal_locale.h` if we are including `pal_localeStringData.h`